doc/sphinx-guides/source/container/dev-usage.rst

```diff
@@ -263,7 +263,7 @@ Hotswapping methods requires using JDWP (Debug Mode), but does not allow switchi
 
         .. image:: img/intellij-compose-add-new-config.png
 
-        Give your configuration a meaningful name, select the compose file to use (in this case the default one), add the environment variable ``SKIP_DEPLOY=1``, and optionally select the services to start.
+        Give your configuration a meaningful name, select the compose file to use (in this case the default one), add the environment variables ``SKIP_DEPLOY=1`` and ``POSTGRES_VERSION=16``, optionally select the services to start.
```

Comment:
The root cause for this is the `.env` file wasn't updated. It still has POSTGRES_VERSION=13 in it. @donsizemore changed the POM, but the .env file change wasn't included.

Reply:
Wondering if we should stop including this file and make it generated by Maven (even without the `-Pct`)

Reply:
Maybe we should start a Chores Github Action that simply collects all of these tedious task that are easy to forget and apply checks like "is the version in these files matching?".

Reply:
Should we bump it? I am just afraid it will hit somehow any non-docker development environment, what do you guys think? @poikilotherm @pdurbin

Reply:
I just saw this on one of the patch-notes "Postgres 13 remains the version used with automated testing." 🤔

Reply:
The .env file is solely used for containers. Safe to upgrade.

Reply:
What do you guys think about this? @poikilotherm @pdurbin Since the guide is shipping with 6.2 this will help that the instructions provided work. In theory, we could just do the previously suggested change for the guide.

Reply:
Developers should probably get used to looking at https://preview.guides.gdcc.io for the latest and greatest. 😄